assets/airports/katl.json

```diff
@@ -943,66 +914,133 @@
+                "GOLLM": ["GOLLM", ["ROHUN", "A240+"], "AZZOG"],
+                "KHMYA": ["KHMYA", "CNDLR", ["FNLEY", "A240+"]],
+                "ORRKK": ["ORRKK", "MORDR", "RAETH", ["NZGUL", "A240+"]],
+                "SHYRE": ["SHYRE", "FRDDO", "BLLBO", ["BGGNS", "A240+"]],
+                "STRDR": [["STRDR", "A240+"], "GIMLY", "AZZOG"]
+            },
+            "body": ["SMAWG", ["GNDLF", "A110+|S250"], ["HALRR", "A100+|S250"], ["SHULR", "A80"], "JNGLE", "^QUBIT", "#005"],
+            "rwy": {
+                "KATL8L": [],
+                "KATL8R": [],
+                "KATL9L": [],
+                "KATL9R": [],
+                "KATL10": []
+            },
+            "draw": [
+                []
+            ]
+        },
+        "HOBTT2": {
+            "icao": "HOBTT2",
+            "name": "Hobbit Two",
+            "entryPoints": {
+                "BEORN": ["BEORN", ["STRDR", "A240+"], "GIMLY", "AZZOG"],
+                "COOUP": ["COOUP", "KHMYA", "CNDLR", ["FNLEY", "A240+"]],
+                "DRSDN": ["DRSDN"],
+                "ENNTT": ["ENNTT", ["GONDR", "A240+"]],
+                "FRDDO": ["FRDDO", "BLLBO", ["BGGNS", "A240+"]],
+                "GOLLM": ["GOLLM", ["ROHUN", "A240+"], "AZZOG"],
+                "KHMYA": ["KHMYA", "CNDLR", ["FNLEY", "A240+"]],
+                "ORRKK": ["ORRKK", "MORDR", "RAETH", ["NZGUL", "A240+"]],
+                "SHYRE": ["SHYRE", "FRDDO", "BLLBO", ["BGGNS", "A240+"]],
+                "STRDR": [["STRDR", "A240+"], "GIMLY", "AZZOG"]
+            },
+            "body": [["SMAWG", "A170+"], ["HOBTT", "A140+"], ["ENSLL", "A120"]],
             "rwy": {
-                "8L": ["GAASS", "KLOWD", "REAKS", "TRAPE", "SNEVE"],
-                "8R": ["GAASS", "KLOWD", "REAKS", "TRAPE", "SNEVE"],
-                "9L": ["FRYES", "SHURT", "STUUF", "SHELE", "SUMNR"],
-                "9R": ["FRYES", "SHURT", "STUUF", "SHELE", "SUMNR"],
-                "10": ["FRYES", "SHURT", "STUUF", "SHELE", "SUMNR"]
+                "KATL26L": ["RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL26R": ["RAIIN", "KLOWD", "SWEPT", "KYMMY", "^KEAVY", "#095"],
+                "KATL27L": ["EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL27R": ["EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"],
+                "KATL28": ["EAGYL", "SHURT", "FOGER", "HITTT", "^YURII", "#095"]
             },
             "draw": [
-                ["DBOLT", "TABBL", "SEETE", "BAAKS", "CHOPI"],
-                ["RAYCL", "SHVRR", "MEEEE", "TMBRZ", "CHOPI"],
-                ["HOWRR", "YORRR", "RIDZE", "NICCE", "CHOPI"],
-                ["EAGYL", "SHELE", "SUMNR"],
-                ["CHOPI", "ONYON", "GAASS"],
-                ["RAIIN", "TRAPE", "SNEVE"],
-                ["ONYON", "FRYES"],
-                ["DDAVS", "TABBL"],
-                ["TRAYE", "TABBL"],
-                ["MMMOE", "YORRR"]
+                []

```

Comment:
Why are we not drawing here? Would be nice for future plans, if STARs are drawn.